sklearn/compose/_column_transformer.py

```diff
@@ -212,6 +214,20 @@ class ColumnTransformer(TransformerMixin, _BaseComposition):
 
     _required_parameters = ["transformers"]
 
+    _parameter_constraints: dict = {
+        "transformers": [list],
```

Comment:
Should we have a `Hidden(tuple)` here to be consistent with `Pipeline`?

Reply:
Actually the `plot_set_output.py` example is using a tuple and this broke the doc build ... I opened #25222 to add `Hidden(tuple)`